packages/loopring_v3/contracts/impl/agents/FastWithdrawalAgent.sol

```diff
@@ -231,16 +231,12 @@ contract FastWithdrawalAgent is ReentrancyGuard
         )
         internal
     {
-        bool success;
-        if (amount > 0) {
-            if (token == address(0)) {
-                // ETH
-                success = to.sendETH(amount, gasleft());
-            } else {
-                // ERC20 token
-                success = token.safeTransferFrom(from, to, amount);
-            }
-        }
+        if (amount == 0) return;
+       
+        bool success = token == address(0) ? 
+            to.sendETH(amount, gasleft()) :
+            token.safeTransferFrom(from, to, amount);
```

Comment:
I think the original code has a bug when account == 0

Reply:
Yes you're right. Though `amount == 0` doesn't do much useful. It could actually just use the verify variants of the functions:
```
if (amount > 0) {
    if (token == address(0)) {
        // ETH
        to.sendETHAndVerify(amount, gasleft());
    } else {
        // ERC20 token
        token.safeTransferAndVerify(to, amount);
    }
}
```